eslint.config.js

```diff
@@ -279,5 +280,21 @@ module.exports = [
     ...eslintPluginYml.configs["flat/recommended"].map(config => ({
         ...config,
         files: [ALL_YAML_FILES]
-    }))
+    })),
+    {
+        name: "eslint/ts-rules",
+        files: ["**/*.ts"],
```

Comment:
should this be limited?
```suggestion
        files: ["tests/lib/types/*.ts"],
```